UnitTest/tests/test_sub_SIGNALduino_CheckCcregResponse-definition.txt

```diff
@@ -1,11 +1,66 @@
 defmod test_sub_SIGNALduino_CheckCcregResponse UnitTest dummyDuino ( 
 {
-	subtest 'Test ccregAll response (ccreg 00: 00  ccreg 10: 57)' => sub {
-		plan tests => 2;
+	subtest 'Test ccregAll response (ccreg:\n 00 \n57\n\nConfiguration Register)' => sub {
+		plan tests => 5;
 		
-		my ($ret)=SIGNALduino_CheckCcregResponse($targetHash,"ccreg 00: 00  ccreg 10: 57");
-		like($ret,qr/^ccreg 00: 00/,"check begin of return message");
-		like($ret,qr/ccreg 10: 57/,"check second part of return message");
+		my ($ret)=SIGNALduino_CheckCcregResponse($targetHash,"ccreg: 
+0D 2E 2D 47 D3 91 3D 04 32 00 00 06 00 21 65 E8
```

Comment:
Ich glaube, das ist nicht der Wert, so wie er vom uC empfangen und an die Routine übergeben wird.

Reply:
Ich habe mich nur an deiner Zeile
`my ($ret)=SIGNALduino_CheckCcregResponse($targetHash,"ccreg 00: 00  ccreg 10: 57");`
orientiert welche natürlich auch so nicht vom uC kommt ;-)

Reply:
Du hast da nicht ganz unrecht, ich hatte es abgekürzt. Bei ccreg 99 (all) kommt:
`ccreg 00: 0D 2E 2D 47 D3 91 3D 04 32 00 00 06 00 10 B0 71  ccreg 10: 57 C4 30 23 B9 00 07 00 18 14 6C 07 00 91 87 6B  ccreg 20: F8 B6 11 EF 0C 3C 1F 41`

Bei ccreg 20 kommt
`C20 = F8`
und etliche
Use of uninitialized value in concatenation (.) or string at ./FHEM/00_SIGNALduino.pm line 1050.

Die Rückgabe alle ohne Zeilenumbrüche natürlich :)